grails-app/assets/javascripts/controllers/modal-video-ctrl.js

```diff
@@ -1,22 +1,49 @@
 'use strict';
 
+/** Directive for formatting from mm:ss to int and back */
+streamaApp.directive("formatDirective", function(){
+  return {
+    restrict: 'A',
+    require: 'ngModel',
+    link: function(scope, element, attrs, ngModelController) {
+      //view to model
+      ngModelController.$parsers.push(function(data) {
+
+       return data.split(":")[0]*60+data.split(":")[1]*1;
+      });
+      //model to view
+      ngModelController.$formatters.push(function(data) {
+        if(data == undefined)
+        {
+          return '';
+        }
+        var seconds = data % 60;
+        var minutes = (data-seconds) / 60;
+        return minutes+':'+seconds;
+      });
+    }
+  };
+});
+
+
 streamaApp.controller('modalVideoCtrl', [
 	'$scope', '$modalInstance', 'apiService', 'video', 'isManual', 'tvShow',
 	function ($scope, $modalInstance, apiService, video, isManual, tvShow) {
 	$scope.loading = false;
+
 	$scope.addManually = isManual;
-		
-	$scope.episode = video || {};
 
+	$scope.episode = video || {};
 	$scope.saveEpisode = function (episode) {
 		if(tvShow)
 			episode.show = tvShow.id;
-		
```

Comment:
You committed too much formatting stuff! This is very bad practice and creates a lot of unnecessary "changes" to review.